### PR TITLE
Update node in docs

### PR DESCRIPTION
**Pull request by aellison**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,8 +73,9 @@ install:
   # Set up front-end dependencies.
   - cd $COGNIZANT_DIR
 
-  - ./install-node.sh 6.10.0
-  - nvm use 6.10.0
+  - nvm ls-remote
```
Comment (skippednote) [inline .travis.yml]: @aellison Do we need to `ls-remote` to install node?

File: STARTERKIT/_readme/setup-full.md [hunk 2]
```diff
@@ -11,16 +11,16 @@ This sub theme uses node and npm to manage build tools such as sass. There are n
 Script usage (from generated sub theme directory):
 
 ```bash
-./install-node.sh 4.4.4
+./install-node.sh 6.11.0
```
Comment (skippednote) [inline STARTERKIT/_readme/setup-full.md]: Do we still need to mention the version when we are using `nvm install lts`?

File: STARTERKIT/_readme/setup-full.md [hunk 3]
```diff
@@ -11,16 +11,16 @@ This sub theme uses node and npm to manage build tools such as sass. There are n
...
 ```
 
 The install script installs nvm and then uses nvm to install the version of node given to it as an argument. It also writes a `.node-version` file into the sub theme directory it was run from that can be picked up by tools such as avn. After running the script it is necessary to do the command it outputs to tell your session you want to use that version of node:
 
 ```bash
-source ~/.bashrc && nvm use --delete-prefix 4.4.4
+source ~/.bashrc && nvm use --delete-prefix 6.11.0
```
Comment (skippednote) [inline STARTERKIT/_readme/setup-full.md]: Do we still need to mention the version when we are using `nvm install lts`?

File: STARTERKIT/_readme/setup-full.md [hunk 4]
```diff
@@ -11,16 +11,16 @@ This sub theme uses node and npm to manage build tools such as sass. There are n
...
 ```
 
-If you are not using avn or something similar then you will need to repeat `nvm use 4.4.4` if you close and reopen your session.
+If you are not using avn or something similar then you will need to repeat `nvm use 6.11.0` if you close and reopen your session.
```
Comment (skippednote) [inline STARTERKIT/_readme/setup-full.md]: Do we still need to mention the version when we are using `nvm install lts`?

File: STARTERKIT/install-node.sh [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 if [ $# -ne 1 ]; then
-    echo $0: usage: install-node.sh 4.4.1
+    echo $0: usage: install-node.sh 6.11.0
```
Comment (skippednote) [inline STARTERKIT/install-node.sh]: Do we still need to mention the version when we are using `nvm install lts`?

**Comment by jenter** [comment]
@aellison since travis needs a version let's just pin to and call this a wrap // 6.11.0

**Comment by skippednote** [comment]
Looks good to merge with `6.11.0`.

**Pull request merged**